include/alpaka/atomic/AtomicOmpBuiltIn.hpp

```diff
@@ -217,21 +219,12 @@ namespace alpaka
         {
             ALPAKA_FN_HOST static auto atomicOp(AtomicOmpBuiltIn const&, T* const addr, T const& value) -> T
             {
+                // TODO(bgruber): atomic increment with wrap around is not implementable in OpenMP 5.1
```

Comment:
I feel that this should be documented somewhere...

Reply:
For alpaka users or OpenMP users? Do you want me to add a documentation comment in alpaka's source?

Reply:
For alpaka users. Does CUDA's atomicInc wrap around? If so, this might lead to unexpected results if a user utilizes both the OpenMP and the CUDA back-end.

Reply:
Yes, CUDA's `atomicInc` wraps around. The OpenMP implementation does the same. It is functionally equivalent. It just uses a lock over an atomic, because we cannot implement it with an atomic.

Reply:
Ah, I see. Then the TODO is fine.

Reply:
ok!